htdocs/install/mysql/migration/13.0.0-14.0.0.sql

```diff
@@ -359,3 +359,6 @@ ALTER TABLE llx_propal ADD CONSTRAINT fk_propal_fk_user_signature FOREIGN KEY (f
 
 UPDATE llx_propal SET fk_user_signature = fk_user_cloture WHERE fk_user_signature IS NULL AND fk_user_cloture IS NOT NULL;
 UPDATE llx_propal SET date_signature = date_cloture WHERE date_signature IS NULL AND date_cloture IS NOT NULL;
+
+INSERT INTO llx_c_action_trigger (code,label,description,elementtype,rang) values ('MEMBER_EXCLUDE','Member excluded','Executed when a member is excluded','member',26);
```

Comment:
If you add into migration, you should also have it into /install/mysql/data/llx_c_action_trigger.sql
Is it ?

Reply:
indeed. I made it!

Reply:
indeed! i made it